anta/custom_types.py

```diff
@@ -85,6 +85,12 @@ def bgp_multiprotocol_capabilities_abbreviations(value: str) -> str:
     BeforeValidator(interface_autocomplete),
     BeforeValidator(interface_case_sensitivity),
 ]
+VxlanSrcIntf = Annotated[
+    str,
+    Field(pattern=r"^(Loopback)[0-9]+$"),
```

Comment:
As loopback support 0-8191 so need to update the patter.

Reply:
Yes and please move this to the test itself. I doubt it will be used elsewhere.

Reply:
Oh never mind! I saw you added extra validators.

Reply:
Updated the custom type. Please take a look once.
@carl-baillargeon